jasmine/spec/factory.js

```diff
@@ -0,0 +1,13 @@
+const file1 = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.'
+  },
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'Alice falls into a rabbit hole and enters a world full of imagination.'
+  }
```

Comment:
Nice one here but you may want to require a file that contains them rather than hard coding.